client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
Should we set `showExternal` to false here if `previewMarkup` is set? Currently the button shows by default but doesn't do anything when markup is used:
![wordpress_com_](https://cloud.githubusercontent.com/assets/416133/13866089/b98aa364-ecfd-11e5-9505-9cc98b3c05a7.png)